src/Stratis.Bitcoin/Consensus/Validators/BlockValidator.cs

```diff
@@ -195,7 +195,7 @@ public async Task<PartialValidationResult> ValidateAsync(Block block, ChainedHea
                 Succeeded = validationContext.Error == null
             };
 
-            this.logger.LogTrace("(-)");
+            this.logger.LogTrace("(-):{0}={1}", nameof(partialValidationResult.Succeeded), partialValidationResult.Succeeded);
```

Comment:
there is override, so you just need `this.logger.LogTrace("(-):{0}", partialValidationResult);`

Reply:
Fixed